apps/spruce/src/components/Table/BaseTable.tsx

```diff
@@ -57,6 +58,9 @@ type SpruceTableProps = {
   loading?: boolean;
   /** estimated number of rows the table will have */
   loadingRows?: number;
+  /** number of total items the table will have */
+  numTotalItems?: number;
+  usePagination?: boolean;
   /** rows that will have a blue tint to represent that they are selected */
   selectedRowIndexes?: number[];
```

Comment:
nit. to be in alphabetical order, `usePagination` should come after `selectedRowIndexes`